graphql/connection_test.go

```diff
@@ -0,0 +1,628 @@
+package graphql_test
```

Comment:
could you separate this commit into two? one that moves existing tests, then one that adds your new tests.

Reply:
done